File: elixir/06-io-operations/read_input.exs
# IO.gets prints the prompt and returns the line (with a trailing newline)
name =
  IO.gets("Enter your name: ")
  |> String.trim()

IO.puts("Hello, #{name}!")

# Input is always text — convert it to work with numbers
age =
  IO.gets("Enter your age: ")
  |> String.trim()
  |> String.to_integer()

IO.puts("Next year you will be #{age + 1}")

File: python/06-io-operations/read_input.py
# Reading input from standard input

# input() prints its prompt, then returns whatever the user types (as a string)
name = input("Enter your name: ")
print(f"Hello, {name}!")

# Convert to int() because input() always returns a string
age = int(input("Enter your age: "))
print(f"Next year you will be {age + 1}")
